nondex-common/src/main/java/edu/illinois/nondex/shuffling/ControlNondeterminism.java

```diff
@@ -60,6 +63,13 @@ public static Configuration getConfiguration() {
         if (originalOrder.size() < 2) {
             return originalOrder;
         }
+
+        initializeNondex();
```

Comment:
Maybe put some comment to remind ourselves why we have to call initializeNondex here even though it is called in static block, that there is this issue with Java 9 classloading.